web/js/notifications/wv.notifications.ui.js

```diff
@@ -58,6 +58,10 @@ wv.notifications.ui = wv.notifications.ui || function(models, config) {
    */
   var init = function() {
     var reactComponent, options, p, alertUser;
+    if(!wv.util.browser.localStorage) {
```

Comment:
Ah, I see. My bad! The rest of my comments weren't relative then. 🙊 I've removed them, but left two other comments.